docs/content/development/extensions-contrib/tdigestsketch-quantiles.md

```diff
@@ -70,49 +68,22 @@ Example:
 }
 ```
 
-|property|description|required?|
-|--------|-----------|---------|
-|type|This String should always be "buildTDigestSketch"|yes|
-|name|A String for the output (result) name of the calculation.|yes|
-|fieldName|A String for the name of the input field containing raw numeric values.|yes|
-|compression|Parameter that determines the accuracy and size of the sketch. Higher compression means higher accuracy but more space to store sketches.|no, defaults to 100|
-
-
-The result of the aggregation is a T-Digest sketch that is built by merging pre-built T-Digest sketches.
-
 ```json
 {
-  "type" : "mergeTDigestSketch",
-  "name" : <output_name>,
-  "fieldName" : <metric_name>,
-  "compression": <parameter that controls size and accuracy>
- }
+	"type": "buildTDigestSketch",
```

Comment:
Since build/merge implementations are combined, suggest calling the type "tDigestSketch"